src/main/resources/db/changelog/changes/01-create-books-table.yaml

```
@@ -0,0 +1,49 @@
+
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: okapustynska
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```

```suggestion
                    primaryKey: true
```
primaryKey is nonnull by default